### DISCUSSION TITLE
Modding

### DISCUSSION THREAD

**Discussion by PittStone** [stohrendorf/CroftEngine#204] [Ideas]
How about Mods for EdisonEngine?  I have a Mod for Tombati that replace Lara with Lara from Third Game. Will this be possible on EdisonEngine? There are some HQ Cutscenes in avi too. Can this work?

**Comment by stohrendorf** [comment]
Modding is currently not possible, but you can replace all media files with other ones as long as you either keep the file name the same (file type shouldn't matter), or if you patch the scripts to refer to the new files (will come in Beta 13).

**Comment by RildSilverlok** [comment]
Implementing Glidos textures is nice . It is pretty cool to see stuff I made almost twenty years ago still being used. 
This might not count as a mod; but if I went back through the texture set and created bump maps for them would/could the bump maps be be implemented?

**Reply by stohrendorf** [reply]
Oh, didn't think you were still active. From all the people who used texture packs, they basically all agree that yours is the best.

Regarding the normal maps, there's currently no technical support implemented, but it would be interesting to see what normal maps would make it look like.

**Comment by RildSilverlok** [comment]
" they basically all agree that yours is the best"...interesting, Perhaps
I  now have a reason to go back and finish them and correct the color
saturation  for non-glidos applications.
Nice work on the croft engine.
